sdk/storagemover/Azure.ResourceManager.StorageMover/api/Azure.ResourceManager.StorageMover.netstandard2.0.cs

```diff
@@ -296,30 +296,61 @@ namespace Azure.ResourceManager.StorageMover.Models
         public static Azure.ResourceManager.StorageMover.Models.StorageMoverAgentPropertiesErrorDetails StorageMoverAgentPropertiesErrorDetails(string code = null, string message = null) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverData StorageMoverData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, System.Collections.Generic.IDictionary<string, string> tags = null, Azure.Core.AzureLocation location = default(Azure.Core.AzureLocation), string description = null, Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState? provisioningState = default(Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState?)) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverEndpointData StorageMoverEndpointData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.ResourceManager.StorageMover.Models.EndpointBaseProperties properties = null) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverProjectData StorageMoverProjectData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string description = null, Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState? provisioningState = default(Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState?)) { throw null; }
     }
+    public partial class AzureKeyVaultSmbCredentials : Azure.ResourceManager.StorageMover.Models.Credentials
+    {
+        public AzureKeyVaultSmbCredentials() { }
+        public string PasswordUri { get { throw null; } set { } }
+        public string UsernameUri { get { throw null; } set { } }
```

Comment:
Change the type to URI

Reply:
Made the type as string as we need to support inputting an empty string for PasswordUri and UsernameUri to clean up the values (specified in https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/storagemover/resource-manager/Microsoft.StorageMover/preview/2023-07-01-preview/storagemover.json#L2160-L2172), while URI doesn't allow an empty string input. 

Is there a way to achieve the cleanup behavior with URI type? Thanks!